### ISSUE TITLE
It's impossible to run two services with stats service enabled

### ISSUE THREAD

**Issue by falkolab** [noctarius/timescaledb-event-streamer#121]
Hi @noctarius!

I use two services running in my environment for the different nats servers. So I realized we have hardcoded port 8081 for statistic service. It would be nice to have it configurable. At this moment I will have to disable it for the one of streamer service instance by using option to prevent port collision:

`PropertyStatsEnabled        = "stats.enabled"`

**Comment by noctarius** [comment]
I'm sorry, but I'm not really sure what you mean with two statistics services? 🤔

**Comment by falkolab** [comment]
I mean I executing two instance of `timescaledb-event-streamer` but both has stats http server listening on port 8081.
So it's impossible to run both without disabling stats on one of them

**Comment by noctarius** [comment]
Ohhhhh I see. That makes sense. Good point, I'll add a config option for it :-)

**Comment by falkolab** [comment]
I have completed some investigations, and as a newbie in NATS and JetStream, I assumed that the stream would be created when publishing a message. However, that is not true. One will have to create the jetstream with wildcard subject for example: `nats str add test_timescaledb_public --subjects "timescaledb.public.*" --max-msgs 1000`

 I would advice some improvements:

1. add configurable deadline on event pushing to not existed stream in order to avoid stuck  
```
// TODO add config param
ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
defer cancel()

_, err = n.jetStreamContext.PublishMsg(
    &nats.Msg{
	    Subject: topicName,
	    Header:  header,
	    Data:    envelopeData,
    },
nats.Context(ctx),
)
```

2. improve the message key name to `Nats-Msg-Id` like mentioned in [nats jetstream docs](https://docs.nats.io/nats-concepts/jetstream/headers). A possible example:
```
	header := nats.Header{}
	hash := md5.Sum(keyData)
	header.Add("Nats-Msg-Id", hex.EncodeToString(hash[:]))
```
What is weird is that the client returned an error `headers not supported by this server` in current key implementation before I had created the stream, and it disappeared after.
